### PR TITLE
install Pillow 2.9

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This fixes issues with Pillow

see output https://gist.github.com/aleksandra-tarkowska/11dd9cc0d8ec7b70f289fe33f7460f21
test training server deployed on http://10.0.51.137

slackscreensnapz001

### CODE REVIEW

**Comment by kennethgillen** [comment]
Certainly looks good. Will test agains infra-testpr.

File: ansible/roles/omero-web-runtime/tasks/main.yml [hunk 1]
```diff
@@ -25,13 +24,15 @@
     state: absent
   with_items:
     - python-gunicorn
+    - python-pillow
```
Comment (joshmoore) [inline ansible/roles/omero-web-runtime/tasks/main.yml]: What's this line for?

**Comment by joshmoore** [comment]
Are there any options for this that would prevent the `gcc` installation?

File: ansible/roles/omero-web-runtime/tasks/main.yml (hunk 1 above)
Comment (aleksandra-tarkowska) [inline ansible/roles/omero-web-runtime/tasks/main.yml]: to remove default one

Comment (joshmoore) [inline ansible/roles/omero-web-runtime/tasks/main.yml]: Ah, missed  "state: absent". Thanks.

**Comment by aleksandra-tarkowska** [comment]
the only option is to give a param to disable scripts. then python-pillow will be chosen. Note pip installed pillow is needed to run script which is pretty much server dependency. I was also planning to refactor and decouple web (as manic suggested omero-python role).

**Comment by manics** [comment]
Have you tested this on a native openstack cento7 image?

**Comment by joshmoore** [comment]
Another question: will any changes in the training playbook be needed re: https://trello.com/c/wEh5uadW/2134-re-do-pillow-on-frog-o-org?

**Comment by aleksandra-tarkowska** [comment]
@joshmoore I am not sure if I understand the question. This PR is to fix training playbook `training-server.yml`. I understand that will affect others, but as it is possible to disable scripts optionally, then person running playbook will be able to decide. I don;t know any other option to install pillow 2.9 without gcc.

**Comment by aleksandra-tarkowska** [comment]
Heads up. If we are not happy for bcc on prod then how you want to pop install zeroc-icepy ? https://github.com/ome/omero-install/blob/develop/linux/step01_centos7_ice_deps.sh#L23

**Comment by joshmoore** [comment]
Think we're into gcc/g++ regardless. Might just be something to make clear upfront.

**Comment by joshmoore** [comment]
This is being migrated to gh-105

**Pull request closed without merging**